jobs/service-fabrik-broker-ext/templates/config/settings.yml.erb

```diff
@@ -0,0 +1,275 @@
+  ####################
+  # GENERAL SETTINGS #
+  ####################
+  broker_name: '<%= broker_name %>'
+  deployment_name: '<%= spec.deployment %>'
+  release_name: '<%= spec.release.name %>'
+  release_version: '<%= spec.release.version %>'
+  username: '<%= link("broker").p('username') %>'
+  password: '<%= link("broker").p('password') %>'
+  enable_service_fabrik_v2: <%= link("broker").p('enable_service_fabrik_v2') %>
+  skip_ssl_validation: <%= link("broker").p('skip_ssl_validation') %>
+  session_store:
+    path: "/var/vcap/store/service-fabrik-broker-ext/session"
+  log_path: <%= log_path %>
+  log_level: <%= link("broker").p('log_level') %>
+  sys_log_level: <%= link("broker").p('sys_log_level') %>
+  enable_bosh_rate_limit: <%= link("broker").p('enable_bosh_rate_limit') %>
+  enable_cross_organization_sharing: <%= link("broker").p('enable_cross_organization_sharing') %>
+  enable_circuit_breaker: <%= link("broker").p('enable_circuit_breaker') %>
+  enable_swarm_manager: <%= link("broker").p('enable_swarm_manager') %>
+  feature:
+    ServiceInstanceAutoUpdate: <%= link("broker").p('feature.ServiceInstanceAutoUpdate') %>
+    EnableSecurityGroupsOps: <%= link("broker").p('feature.EnableSecurityGroupsOps') %>
+    AllowInstanceSharing: <%= link("broker").p('feature.AllowInstanceSharing') %>
+  http_timeout: <%= link("broker").p('http_timeout') %>
+  deployment_action_timeout: <%= link("broker").p('deployment_action_timeout') %>
+  agent_operation_timeout: <%= link("broker").p('agent_operation_timeout') %>
+  multi_az_enabled: <%= link("broker").p('multi_az_enabled') %>
+  ha_enabled: <%= link("broker").p('ha_enabled') %>
+  broker_drain_message: <%= link("broker").p('broker_drain_message') %>
+  ##############################
+  # EXTERNAL ENDPOINT SETTINGS #
+  ##############################
+  external:
+    log_event: <%= link("broker").p('external.log_event') %>
+    event_type: SF.API_EVENT
+    trust_proxy: <%= external['trust_proxy'] %>
+    port: <%= external['port'] %>
+    protocol: 'https'
+    <% link("broker").if_p('external.ssl') do |ssl| %>
+    ssl: <%= JSON.dump(ssl) %>
+    <% end %>  
+    host: <%= external['host'] %>
+    cookie_secret: <%= external['cookie_secret'] %>
+    cookie_secure: true
+    session_expiry: <%= external['session_expiry'] %>
+    api_requires_admin_scope: <%= link("broker").p('external.api_requires_admin_scope', false) %>
+
+  ##############################
+  # INTERNAL ENDPOINT SETTINGS #
+  ##############################
+  internal:
+    log_event: <%= link("broker").p('internal.log_event') %>
+    event_type: SF.BROKER_EVENT
+    domain_socket:
+      path: <%= link("broker").p('internal.domain_socket.path') %>
+      fwd_events: true
+    port: <%= internal['port'] %>
+    ip: <%= internal['ip'] %>
+    host: <%= "#{internal['ip']}:#{internal['port']}" %>
+    protocol: 'https'
+    <% link("broker").if_p('internal.ssl') do |ssl| %>
+    ssl: <%= JSON.dump(ssl) %>
+    <% end %>
```

Comment:
Do we still need internal endpint settings for this job?

Reply:
kept all properties for now, these can be removed one by one.